build/package/entrypoint.sh

```diff
@@ -1,8 +1,9 @@
 #!/bin/sh
+set -e
 
-if [ "$(echo $1 | head -c 1)" != "-" ] ; then
+if [ $# != 0  ] ; then
   exec "$@"
```

Comment:
this would allow running any command, it's fine but not good.

this image does not like other base images - such as alpine, ubuntu. this image is dedicated to easegress.

So, it's better for people can easy add the easegress parameters in command line instead ask user run something like this:

```
docker run easegress /opt/easegress/bin/easegress-serve foo bar
```

the better way should be:

```
docker run easegress easegress  foo bar
```

Please check the Docker official best practice - [EntryPoint](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#entrypoint) , and take a look the postgress example.

Reply:
* The original docker image file can let user run default configuration with 
`docker run easegress --`
  but it supports adding param in this format
`dock run easegress --foo --bar`
* My first consideration is to support running in 
`docker run easegress` with default configuration. And yes, `docker run easegress /opt/easegress/bin/easegress-serve foo bar` is a bit weird, let me align with other popular product's image

Reply:
@haoel 
Update it according to the best docker best practice guide.
Now we support 
``` bash
docker run -it  megaease/easegress:latest  easegress-sever --debug  // with whatever command u want

docker run -it  megaease/easegress:latest  --debug   // only with easegress parameters

docker run -it  megaease/easegress:latest    // with all default parameters
```